docs/root/manual/developer-docs/nodes/mithril-signer.md

```diff
@@ -152,6 +169,14 @@ If you want to dig deeper, you can get access to several level of logs from the
 
 :::
 
+## Download compiled binary
+
+Each Release / Pre-Release distribution comes with pre compiled binaries ready to use or wrapped in a debian package.
+
+You can download them from the Release / Pre-Release distribution page that depends on the Mithril Network you are targeting.
+
+These links are available in the **Build From** column of the above **Mithril Networks** table.
```

Comment:
Instead of repeating those blocks every time don't you want to do a shared block similar to the network matrix ?

Reply:
Done :heavy_check_mark: